cluster-setup/upgrade-test-cluster/performance/performance_profile.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: performance.openshift.io/v1alpha1
+apiVersion: performance.openshift.io/v1
```

Comment:
upgrade test failed with error:
`error: unable to recognize "STDIN": no matches for kind "PerformanceProfile" in version performance.openshift.io/v1" `

In upgrade test workflow we are using this yaml file for creating profile and or some reason v1 does not know about PerformanceProfile objects..

Reply:
How does the test work? What is the first profile we update from? I will look in order to understand. All other e2e tests are passing, so there is something I might have missed.

Reply:
I think the problem is the "InitialProfile" is from 4.5, meaning v1alpha1, so if the test is asking for a v1 profile BEFORE the upgrade, it will fail.

Reply:
Looking at the prowl yaml, the base image is: ee686d6 which is BEFORE the PR and the job runs e2e-gcp-operator-upgrade, so is it possible we start with a prev PAO (before the PR code) ?

Reply:
Resolved, thanks!